src/sc_generation/generation.rs

```diff
@@ -19,29 +19,16 @@ pub fn generate_op_datastore() -> Datastore {
     let mut datastore: Datastore = Datastore::new();
     let nb_entries = 100;
     for _ in 0..nb_entries {
-        unsafe {
-            let key = generate_string(rng.gen_range(5..32))
-                .encode_utf16()
-                .collect::<Vec<u16>>()
-                .align_to::<u8>()
-                .1
-                .to_vec();
-            let value = generate_string(rng.gen_range(5..100))
-                .encode_utf16()
-                .collect::<Vec<u16>>()
-                .align_to::<u8>()
-                .1
-                .to_vec();
-            datastore.insert(key, value);
+        /*unsafe*/
```

Comment:
you can prevent `jump scare` by removing `unsafe` even if it's commented :D

Reply:
Done